src/shared_modules/utils/socketDBWrapper.hpp

```diff
@@ -56,10 +56,10 @@ class SocketDBWrapper final : public Singleton<SocketDBWrapper>
     bool m_dataReady {false};
 
 public:
-    void init()
+    void init(const std::string& socketPath = WDB_SOCKET)
```

Comment:
Why is the socket path parameter if this is only for WDB?

Reply:
This make the use of the testtool not possible, we can revert this changes